curated_transformers/tests/models/test_hf_model.py

```diff
@@ -76,12 +76,18 @@ def test_model_against_hf_transformers(model_config):
     )
     Y_hf_encoder = hf_encoder(X, attention_mask=attention_mask)
 
-    assert torch.allclose(
-        Y_encoder.last_hidden_layer_states, Y_hf_encoder.last_hidden_state, atol=1e-6
+    torch.testing.assert_close(
+        Y_encoder.last_hidden_layer_states,
+        Y_hf_encoder.last_hidden_state,
+        atol=1e-05,
+        rtol=1e-05,
```

Comment:
Won't the default tolerances work? See the table at:

https://pytorch.org/docs/stable/testing.html#torch.testing.assert_close

I am kinda surprised that the default `rtol` would be too strict. Could this be an issue because one tensor is a CuPy tensor and it will switch to the tolerances `0.0`/`0.0` due to a `dtype` mismatch? If so, it'd probably be better to convert our tensor to Torch (or the Huggingface tensor to `ops.xp`) and use the defaults.

Reply:
Hmmm, that's a shame. Maybe we should make constants for the default tolerances (or a wrapper function), since most are the same that's probably a little cleaner than having `atol`/`rtol` everywhere. If one case deviates, we could always state the tolerances explicitly there.